app.js

```diff
@@ -24,17 +25,21 @@ app.get('/', function get(req, res) {
   }));
 });
 
-app.get('/responses', function get(req, res) {
-return smartSurveyAPICall.getData(
-    config.apiToken,
-    config.apiTokenSecret,
-    config.surveyID)
-  .then(results => res.json(results))
-  // This is a placeholder for error logging. In the future, the aim
-  // is to print a friendly message to the user with the stacktrace
-  // eslint-disable-next-line no-console
-  .catch(error => console.log(error));
-});
+// Create a function that passes in a var rather than having var in the function
+// itself.  Otherwise you have a dependency of var especially if it's a config.
+// Therefore, you'll have to stub it
+function createGet(surveyID) {
+  return function get(req, res) {
+    return smartSurveyAPI.getData(surveyID)
+      .then(results => res.json(results))
+      // This is a placeholder for error logging. In the future, the aim
+      // is to print a friendly message to the user with the stacktrace
+      // eslint-disable-next-line no-console
+      .catch(error => console.log(error));
```

Comment:
```js
.catch(console.log);
```
Unless I'm going mad...

Reply:
It did work but I get this in the log `(node:18699) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: res.json(...).catch is not a function`

Reply:
What's the exact code that's causing that? It sounds unrelated.

Reply:
You could be right that it's unrelated.  

I'm not sure where the exact code is, it could be the way res.json works or 

could how I'm dealing with the Promise here

https://github.com/UKHomeOffice/mapping-participants/pull/7/files#diff-7a540b8500a88da5e725a07b8a9ba1fcR30

or here

https://github.com/UKHomeOffice/mapping-participants/pull/7/files#diff-0364f57fbff2fabbe941ed20c328ef1aR33